source/state_representation/src/trajectory/CartesianTrajectory.cpp

```diff
@@ -0,0 +1,131 @@
+#include "state_representation/trajectory/CartesianTrajectory.hpp"
+#include "state_representation/space/cartesian/CartesianState.hpp"
+
+#include "state_representation/exceptions/IncompatibleReferenceFramesException.hpp"
+
+namespace state_representation {
+CartesianTrajectory::CartesianTrajectory() {
```

Comment:
You don't need to call the parent constructor?

Reply:
in this case I just saw it as extra lines of code that don't differ. We might be more flexible by separating those 2 in case we want to do something else in the empty constructor, but for the time being I don't see the reason to separate them really